ksql-serde/src/test/java/io/confluent/ksql/serde/connect/ConnectDataTranslatorTest.java

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.serde.connect;
+
+import io.confluent.ksql.GenericRow;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.errors.DataException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class ConnectDataTranslatorTest {
+  final ConnectDataTranslator connectToKsqlTranslator = new ConnectDataTranslator();
+
+  @Test
+  public void shouldTranslateStructCorrectly() {
+    final Schema structSchema = SchemaBuilder
+        .struct()
+        .field("INT", SchemaBuilder.INT32_SCHEMA)
+        .field("LONG", SchemaBuilder.INT64_SCHEMA)
```

Comment:
`BIGINT` instead of `LONG`.

Reply:
This is just a field name. But I'll change it.